Userland/Libraries/LibWeb/HTML/HTMLSelectElement.cpp

```diff
@@ -42,6 +42,27 @@ JS::GCPtr<HTMLOptionsCollection> const& HTMLSelectElement::options()
     return m_options;
 }
 
+// https://html.spec.whatwg.org/multipage/form-elements.html#dom-select-length
+size_t HTMLSelectElement::length()
+{
+    // The length IDL attribute must return the number of nodes represented by the options collection. On setting, it must act like the attribute of the same name on the options collection.
+    return const_cast<HTMLOptionsCollection&>(*options()).length();
```

Comment:
These const casts are a bit ugly, can we just make the getter return a non-const reference? No need to pretend const correctness only to throw it out the window a few lines later.